optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/attributes_opta.adoc

```diff
@@ -0,0 +1,4 @@
+
+
+:PLANNER: OptaPlanner
+:PRODUCTVERSION:7.38.0.Final
```

Comment:
Hard-coded version. This way, it needs to be manually updated and committed with each project version change.

Better approach described here: https://asciidoctor.org/docs/asciidoctor-maven-plugin/#add-version-and-build-date-to-the-header.

So in `optaplanner-docs/pom.xml` in `asciidoctor-maven-plugin`'s `<configuration>`:
```xml
<configuration>
  ...
  <attributes>
    <PRODUCTVERSION>${project.version}</PRODUCTVERSION>
  </attributes>
</configuration>
```